bool check
---
int32 tileX